packages/ui/__stories__/NumberBadge.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { NumberBadge } from '../src/NumberBadge'
+
+export default {
+  title: 'components/NumberBadge',
+  component: NumberBadge,
+}
+
+export const Default = () => <NumberBadge number={1} />
+
+export const LessThan99 = () => <NumberBadge number={99} />
+
+export const MoreThan99 = () => <NumberBadge number={100} />
```

Comment:
nit: How about this?
```suggestion
export const OneDigit = () => <NumberBadge number={1} />

export const TwoDigits= () => <NumberBadge number={99} />

export const ThreeDigits = () => <NumberBadge number={100} />
```